src/utils/message-popup.js

```diff
@@ -1,8 +1,49 @@
 import { Modal } from 'antd';
 
 // displays a modal with a message to the user
-export const messagePopup = message => {
-  return Modal.warning({
-    content: message,
-  });
+// pass in a type of dialog box to display and the message and optional title
+// types are: info, success, error or warning
+export const messagePopup = (type, message, title) => {
+  switch (type) {
```

Comment:
Looks great.. I wonder if instead of using a switch, if you could just plug the 'type' prop straight in. Just a thought for later refactor. I could be wrong about that, but it looks like it would work. Nice job man!